backend/app/migrations/0001_initial_schema.sql

```diff
@@ -1,13 +1,15 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
Postgres has a built in uuid type, why do we load uuid-ossp?

https://www.postgresql.org/docs/current/functions-uuid.html#FUNCTIONS-UUID provides uuid version 4